packages/components/typography/stories/body.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { Body as BodyComponent } from "../src";
+import { ThemeProvider } from "styled-components";
+import { theme } from "@buoysoftware/anchor-theme";
+
+export default {
+  title: "Typography / Components / Body",
+  component: BodyComponent,
+  argTypes: {
+    as: {
+      options: ["p", "span"],
```

Comment:
Should div be an option here? (by default the text will be in a div unless otherwise specified since you are using Element as the base

Reply:
Fair enough.